src/ray/gcs/call_back.h

```diff
@@ -0,0 +1,30 @@
+#ifndef RAY_GCS_CALL_BACK_H
+#define RAY_GCS_CALL_BACK_H
+
+#include <boost/optional/optional.hpp>
+#include <vector>
+#include "ray/common/status.h"
+
+namespace ray {
+
+namespace gcs {
+
+using StatusCallback = std::function<void(Status status)>;
+
+template <typename Data>
+using OptionalItemCallback =
+    std::function<void(Status status, boost::optional<Data> datum)>;
```

Comment:
typo: `datum` ? same for below

Reply:
> typo: `datum` ? same for below

:) my fault, i thought datum has the same means  as data.